webapp/src/shared/components/Button/Button.css

```diff
@@ -1,6 +1,6 @@
 .button {
-  width: 312px;
-  height: 58px;
+  width: 15rem;
+  height: 3rem;
   border-radius: 8px;
```

Comment:
shouldn't we also convert the 8px radius to rem?

Reply:
Updated!